echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/bearychat/CreateP2PChannelResponse.groovy

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Netflix, Inc.
```

Comment:
These copyrights should be for the year 2018, and instead of Netflix either use your name, or your companies name (depending on contracts/legal constraints).